requirements/base.txt

```diff
@@ -13,4 +13,6 @@ ordered-set==2.0.1                  # MIT
 Markdown==2.6.6    					# BSD
 
 -e git+https://github.com/edx/opaque-keys.git@d45d0bd8d64c69531be69178b9505b5d38806ce0#egg=opaque-keys
```

Comment:
While we are at it, this package is also up on pypi: https://pypi.python.org/pypi/edx-opaque-keys/0.3.4